Library/Formula/makemake.rb

```diff
@@ -0,0 +1,15 @@
+class Makemake < Formula
+  homepage "https://github.com/Julow/makemake"
```

Comment:
Could you add a short description, using `desc`? The string `makemake: <description>` should be under 80 chars. Thanks!

For example:

```ruby
desc "Makefile generator"
```